mirror/github/utils.py

```diff
@@ -0,0 +1,87 @@
+import csv
+import json
+import os
+import click
+from pathlib import Path
+import pandas as pd
+
+
+
+
+
+@click.command()
+@click.option('--path_input_folder', '-p', default='.', help='folders with repo/commits. default="." ')
```

Comment:
`--path_input_folder` is not a good argument name. It doesn't match our conventions anyway. `--path-input-folder` is an improvement, but I would prefer a shorter name.